blog-service/2024-12-24-apps.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Apps, Solutions, and Collection Integrations - December Release (Observability)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - apps
+  - releases-notes
+hide_table_of_contents: true  
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+### New release
+
+We’re excited to announce the release of new Azure Service Bus, Azure API Management, and AWS Serverless Application Models for Sumo Logic.
+
+- **Azure Service Bus**. Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, and resource usage of your Service Bus namespace. [Learn more](/docs/integrations/microsoft-azure/azure-service-bus/).
+
+- **Azure API Management**. Azure API Management is a hybrid multicloud platform for managing APIs across different environments. As a platform-as-a-service, it supports the entire API lifecycle and provides near real-time visibility into API status and health, offering essential API Management operations and details for auditing. [Learn more](/docs/integrations/microsoft-azure/azure-api-management/).
+
+- **AWS Serverless Application Models**. Released the following four SAM’s with Python v3.13 AWS Lambda runtime update with latest libraries.
+
+    - sumologic-securityhub-connector-aws-org - SAM SemanticVersion: 1.0.8.
+
+    - sumologic-s3-logging-auto-enable - SAM SemanticVersion: 1.0.15.
+
+    - sumologic-aws-cloudtrail-benchmark - SAM SemanticVersion: 1.0.18.
+
+    - sumologic-app-utils - SAM SemanticVersion: 2.0.19.
+
+### Enhancements 
+
+- We're excited to announce the release of the enhancements listed below for the Sumo Logic apps:
+
+    - **IIS - OpenTelemetry**. Added 10 new monitors for IIS - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/web-servers/iis-10).
```

Comment:
```suggestion
    - **IIS - OpenTelemetry**. Added 10 new monitors that will be triggered for different pre-defined conditions. [Learn more](https://help.sumologic.com/docs/integrations/web-servers/iis-10).
```

Reply:
Follow the same for other apps listed in this list. This is just to avoid using the app name multiple times.